ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -277,156 +241,32 @@ export const Homepage = withUserProfile()(class extends React.Component<
       }
     ];
 
-    if (environment.enableHomepageRestyle) {
```

Comment:
GitHub is making this diff look more "interesting" than it really is. In this if-else statement here, I kept everything within the "if environment.enableHomepageRestyle" and removed everything in the "else".